sencha-workspace/SlateTasksTeacher/app/store/StudentTasks.js

```diff
@@ -0,0 +1,29 @@
+Ext.define('SlateTasksTeacher.store.StudentTasks', {
+    extend: 'Ext.data.Store',
+    requires: [
+        'Slate.cbl.model.StudentTask'
```

Comment:
@nbey this is redundant, stores add their `model` to the requires list automatically.